### ISSUE TITLE
Incorrect quickinfo tooltip on types/namespaces in some cases

### ISSUE THREAD

**Issue by cpyrgas** [X-Sharp/XSharpPublic#1176]
In the following, hover the mouse over the words "Collections", "ArrayList", "INT", "OBJECT" in the last 3 lines, this will always get a correct tooltip showing info about the type or namespace.

But do the same also for the first two LOCAL declarations, now the tooltip says "UNDECLARED Collections", "UNDECLARED BinaryReader" etc

```
FUNCTION Start() AS VOID STRICT
    LOCAL r AS System.IO.BinaryReader
    LOCAL a AS System.Collections.ArrayList
    LOCAL n AS INT
    LOCAL o AS OBJECT
    o := (System.Collections.ArrayList)o
```

**Comment by RobertvanderHulst** [comment]
Fixed in 0adec77e03ef0c5bcce01907d4467de9f07dc0da

**Comment by cpyrgas** [comment]
Confirmed fixed
